src/style.css

```diff
@@ -14,4 +29,176 @@
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  position: relative;
+}
+
+/* mobile-first */
+
+.popup {
+  display: none;
+  width: 100%;
+  height: 105vh;
+  overflow: scroll;
+  position: fixed;
+  padding: 15%;
+  z-index: 100;
+  background-color: burlywood;
+  margin-top: -5%;
```

Comment:
- [ ] This snippet of code is actually repeated also in the class `popup2` so kindly remove it, because you will get all these properties from the `popup2` class.